pkg/ibmvpc/examples/out/explain_out/tgwEnabledSpecificFilter_all_vpcs_explain_debug.txt

```diff
@@ -1,5 +1,5 @@
-Connectivity explanation between ky-vsi1-subnet20 and ky-vsi0-subnet2 within combined-vpc-local-tg-ky
```

Comment:
missing the prefix vpc before the vsi name? ( e.g. `(vpc-name)/ky-vsi1-subnet20` ) ?

Reply:
Its not "missing" since its not part of the name or of the input. In case of `multi-vpc` we can add the name of the relavant `vpc` for `src/dst` of the header.
Perhaps we can change the Name() function in case of multi-vpc to include the vpc?

Reply:
ok, let's open another issue for this to be discussed and then decide.

Reply:
#521